Bug: Ferdium only shows blank window

### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues) for a bug report similar to mine, in vain

### Ferdium Version

6.0.0.nightly.56.r5506.cf182890-1 (AUR ferdi-git package)

### What Operating System are you using?

Other Linux

### Operating System Version

Archlinux

### What arch are you using?

x64

### Last Known Working Ferdium version

_No response_

### Expected Behavior

When starting ferdium, the window should show my services and I should be able to interact with them.

### Actual Behavior

After starting (and opening) ferdium, the window remains blank (see screenshot below).
I get the following log output:

### Steps to reproduce

Launch ferdium

### Debug link

Not Possible. The help menu has no entries (show `(empty)`

### Screenshots

Screenshot of the ferdium window, just to make sure we talk about the same thing when I say it remains blank
![Screenshot_20220611_132620](https://user-images.githubusercontent.com/20126509/173185898-4618f7ea-f18c-4e4e-9331-ee214bb1316c.png)


### Additional information

```bash
$ ferdium --enable-logging             
[37145:0611/132530.385413:WARNING:bluez_dbus_manager.cc(248)] Floss manager not present, cannot set Floss enable/disable.
[37177:0611/132530.411115:WARNING:sandbox_linux.cc(376)] InitializeSandbox() called with multiple threads in process gpu-process.
[37145:0611/132530.644542:INFO:CONSOLE(111)] "%cDownload the React DevTools for a better development experience: https://reactjs.org/link/react-devtools
You might need to use a local HTTP server (instead of file://): https://reactjs.org/link/react-devtools-faq font-weight:bold", source: /opt/ferdium-git/resources/app.asar/node_modules/@sentry/utils/dist/instrument.js (111)
[37145:0611/132531.169996:INFO:CONSOLE(943)] "Uncaught Error: Cannot find module 'react-virtualized'
Require stack:
- /opt/ferdium-git/resources/app.asar/node_modules/electron-react-titlebar/dist/renderer.js
- /opt/ferdium-git/resources/app.asar/components/layout/AppLayout.js
- /opt/ferdium-git/resources/app.asar/containers/layout/AppLayoutContainer.js
- /opt/ferdium-git/resources/app.asar/routes.js
- /opt/ferdium-git/resources/app.asar/index.html", source: node:internal/modules/cjs/loader (943)
[37145:0611/132532.337313:INFO:CONSOLE(111)] "Could not connect to Sentry main process. Did you call init in the Electron main process?", source: /opt/ferdium-git/resources/app.asar/node_modules/@sentry/utils/dist/instrument.js (111)
```

This is the error I see on my mac with build 57.
Uncaught Error: Cannot find module 'react-virtualized'
```
Require stack:
- /Applications/Ferdium.app/Contents/Resources/app.asar/node_modules/electron-react-titlebar/dist/renderer.js
- /Applications/Ferdium.app/Contents/Resources/app.asar/components/layout/AppLayout.js
- /Applications/Ferdium.app/Contents/Resources/app.asar/containers/layout/AppLayoutContainer.js
- /Applications/Ferdium.app/Contents/Resources/app.asar/routes.js
- /Applications/Ferdium.app/Contents/Resources/app.asar/index.html
    at Module._resolveFilename (node:internal/modules/cjs/loader:940:15)
    at i._resolveFilename (node:electron/js2c/renderer_init:33:1095)
    at Module._load (node:internal/modules/cjs/loader:785:27)
    at c._load (node:electron/js2c/asar_bundle:5:13343)
    at i._load (node:electron/js2c/renderer_init:33:356)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at menu-list-item.tsx:4:108
    at Object.<anonymous> (/Applications/Ferdium.app/Contents/Resources/app.asar/node_modules/electron-react-titlebar/dist/renderer.js:1:8591)
    at Object.<anonymous> (/Applications/Ferdium.app/Contents/Resources/app.asar/node_modules/electron-react-titlebar/dist/renderer.js:3:3)```

node:internal/modules/cjs/loader:943

--- a/package.json
+++ b/package.json
@@ -70,7 +70,7 @@
     "debug": "4.3.4",
     "electron-dl": "^3.3.1",
     "electron-find": "^1.0.7",
-    "electron-react-titlebar": "^1.1.0",
+    "electron-react-titlebar": "1.1.0",
     "electron-updater": "^4.6.5",
     "electron-window-state": "^5.0.3",
     "fast-folder-size": "^1.6.1",

